geode-core/src/main/java/org/apache/geode/management/internal/web/controllers/ShellCommandsController.java

```diff
@@ -97,7 +108,7 @@
   private ResponseEntity<InputStreamResource> getJsonResponse(String result) {
     HttpHeaders respHeaders = new HttpHeaders();
     InputStreamResource isr;// if the command is successful, the output is the filepath,
-    // else we need to send the orignal result back so that the receiver will know to turn it
+    // else we need to send the original result back so that the receiver will know to turn it
```

Comment:
The comment formatting could be made clearer. It's awkward to read jammed into the preceding declaration with no spacing.

Reply:
I agree!  I try to adjust them when I see things like this, but this one must've flown under my radar.